### PR TITLE
Document dropdown menu

### PR DESCRIPTION
**Pull request by pbugnion**
This PR documents the DropdownMenu component. It builds off PR #64  .

This is what the documentation looks like:

dbc-dropdown-menu

### CODE REVIEW

**Comment by tcbegley** [comment]
This is already looking great! 

Looks like the `size` prop isn't currently exposed in the Python API for `DropdownMenu`, this would be a good place to add it I guess. Should we use `bs_size` as in `Input` and `DatePicker*` for consistency?

There's also something pretty strange happening with the dropup, seems to be a problem with the underlying component itself.

**Comment by pbugnion** [comment]
> Looks like the size prop isn't currently exposed in the Python API for DropdownMenu, this would be a good place to add it I guess. Should we use bs_size as in Input and DatePicker* for consistency?

Agreed. I've done this in PR #61 , which this is now rebased on top of.

> There's also something pretty strange happening with the dropup, seems to be a problem with the underlying component itself.

Well spotted. I had a brief look at this and it wasn't completely obvious what we were doing wrong (if anything). I've opened issue #62 to track this.

**Comment by pbugnion** [comment]
I think this is now ready for review.

File: src/components/DropdownMenu.js [hunk 1]
```diff
@@ -80,17 +82,17 @@ DropdownMenu.propTypes = {
...
-   * Label for the toggle
+   * Label for the toggle.
    */
   label: PropTypes.string,
 
   /**
-   * Direction in which to expand the dropdown.
+   * Direction in which to expand the dropdown. Default: 'down'.
    */
   direction: PropTypes.oneOf(['up', 'down', 'left', 'right']),
```
Comment (tcbegley) [inline src/components/DropdownMenu.js]: Shall we change this to just 'down', 'left' or 'right' as discussed in #62 as part of this PR?

**Comment by tcbegley** [comment]
This looks great! The only question I have is whether we should implement the temporary fix for #62 as part of this PR as commented inline.

**Comment by pbugnion** [comment]
> The only question I have is whether we should implement the temporary fix for #62 as part of this PR as commented inline.

That definitely makes sense. I've pushed PR #64 separately and rebased this on top of that.

**Comment by tcbegley** [comment]
Looks great!

**Pull request merged**